app/views/contexts/section_summary_context.py

```diff
@@ -137,46 +156,46 @@ def _list_summary_element(self, summary, current_location, section) -> Mapping:
                 "id"
             ]
 
-        add_link = self._add_link(
-            summary, current_location, section, list_collector_block
-        )
-
-        list_context = ListContext(
-            self._language,
-            self._schema,
-            self._answer_store,
-            self._list_store,
-            self._progress_store,
-            self._metadata,
-        )
+        add_link = self._add_link(summary, section, list_collector_block)
 
         rendered_summary = self._placeholder_renderer.render(
             summary, current_location.list_item_id
         )
 
+        list_summary_context = {}
+
+        if list_collector_block:
+            list_context = ListContext(
```

Comment:
The context should be conditional on the list collector block as you may still have a primary person list collector.

Reply:
What is the scenario for this, and does it need a test given the tests are passing?

Reply:
As discussed, this is for when you have a primary person with no list collectors on the path. We should add a test for it but could be seperate.

Reply:
Added a test and new test schema. Thanks!

Reply:
This should be conditional.